runtime/frequency/src/lib.rs

```diff
@@ -330,7 +330,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("frequency"),
 	impl_name: create_runtime_str!("frequency"),
 	authoring_version: 1,
-	spec_version: 75,
+	spec_version: 78,
```

Comment:
Should get merged after
https://github.com/LibertyDSNP/frequency/pull/1956 and
https://github.com/LibertyDSNP/frequency/pull/1944

Reply:
1956 is merging into a dev branch, but agreed on 1944. Either way, there isn't a requirement to only bump by 1, so you're good.